README.md

```diff
@@ -1,23 +1,23 @@
-# Reaction Admin
+# Admin
 
-[![Circle CI](https://circleci.com/gh/reactioncommerce/reaction-admin.svg?style=svg)](https://circleci.com/gh/reactioncommerce/reaction-admin) [![Gitter](https://badges.gitter.im/JoinChat.svg)](https://gitter.im/reactioncommerce/reaction?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+[![Circle CI](https://circleci.com/gh/reactioncommerce/reaction.svg?style=svg)](https://circleci.com/gh/reactioncommerce/reaction) [![Gitter](https://badges.gitter.im/JoinChat.svg)](https://gitter.im/reactioncommerce/reaction?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
```

Comment:
I think you want to get rid of reaction here not admin.

Reply:
Haha yes, thank you.